src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
Consider destructuring 'person' props in the parameter for a cleaner look.

Reply:
Not fixed 
![image](https://github.com/mate-academy/react_person/assets/99403198/1bd8bcbd-1b86-4141-82cc-6b4f89989cab)

Use destructuring for the person object.
```
const { name, age ... } = person;

Reply:
+